```
make[1]: *** [Javadoc.gmk:718: /tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/build/linux-x86_64-normal-server-release/docs/jre/api/security/jaas/spec/index.html] Killed
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [Javadoc.gmk:569: /tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/build/linux-x86_64-normal-server-release/docs/jre/api/plugin/dom/index.html] Killed
make[1]: *** [Javadoc.gmk:1240: /tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/build/linux-x86_64-normal-server-release/docs/jre/api/accessibility/jaccess/spec/index.html] Killed
make[1]: *** [Javadoc.gmk:983: /tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/build/linux-x86_64-normal-server-release/docs/jdk/api/attach/spec/index.html] Killed
make[1]: *** [Javadoc.gmk:1139: /tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/build/linux-x86_64-normal-server-release/docs/jdk/api/nashorn/index.html] Killed
make[1]: *** [Javadoc.gmk:1085: /tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/build/linux-x86_64-normal-server-release/docs/jdk/api/javac/tree/index.html] Killed
/tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/jdk/src/share/classes/com/sun/jdi/InterfaceType.java:193: warning - Tag @link: can't find INVOKE_SINGLE_THREADED in com.sun.jdi.InterfaceType
/tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00/jdk/src/share/classes/com/sun/management/HotSpotDiagnosticMXBean.java:52: warning - Tag @see: reference not found: ManagementFactory#getPlatformMXBeans(Class)
1 warning
1 warning
make: *** [/tmp/nix-build-openjdk-8u122b00.drv-0/jdk8u-jdk8u122-b00//make/Main.gmk:160: docs-only] Error 2
builder for â€˜/nix/store/qg9vf4w119xw7rmjd7sxapjmqhynxr5k-openjdk-8u122b00.drvâ€™ failed with exit code 2
```
Priceless irony :)

--- a/pkgs/development/tools/misc/bsdbuild/default.nix
+++ b/pkgs/development/tools/misc/bsdbuild/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, perl, libtool, pkgconfig, gettext, groff, ed }:
+{ stdenv, fetchurl, perl, libtool, pkgconfig, gettext, mandoc, ed }:
 
 stdenv.mkDerivation rec {
   name = "bsdbuild-${version}";
@@ -9,7 +9,7 @@ stdenv.mkDerivation rec {
     sha256 = "1zrdjh7a6z4khhfw9zrp490afq306cpl5v8wqz2z55ys7k1n5ifl";
   };
 
-  buildInputs = [ perl groff ed ];
+  buildInputs = [ perl mandoc ed ];
   nativeBuildInputs = [ pkgconfig libtool gettext ];
 
   prePatch = ''

--- a/pkgs/tools/misc/mandoc/default.nix
+++ b/pkgs/tools/misc/mandoc/default.nix
@@ -1,0 +1,35 @@
+{ stdenv, fetchurl, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "mandoc-${version}";
+  version = "1.13.4";
+
+  src = fetchurl {
+    url = "http://mdocml.bsd.lv/snapshots/mdocml-${version}.tar.gz";
+    sha256 = "1vz0g5nvjbz1ckrg9cn6ivlnb13bcl1r6nc4yzb7300qvfnw2m8a";
+  };
+
+  buildInputs = [ zlib ];
+
+  configureLocal = ''
+    HAVE_WCHAR=1
+    MANPATH_DEFAULT="/run/current-system/sw/share/man"
+    OSNAME="NixOS"
+    PREFIX="$out"
+    HAVE_MANPATH=1
+    LD_OHASH="-lutil"
+    BUILD_DB=0
+  '';
+
+  preConfigure = ''
+    echo $configureLocal > configure.local
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "http://mdocml.bsd.lv/";
+    description = "suite of tools compiling mdoc and man";
+    license = licenses.bsd3;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ ramkromberg ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -2565,6 +2565,8 @@ in
 
   man-db = callPackage ../tools/misc/man-db { };
 
+  mandoc = callPackage ../tools/misc/mandoc { };
+
   mawk = callPackage ../tools/text/mawk { };
 
   mbox = callPackage ../tools/security/mbox { };

